packages/ts/react-crud/src/autoform.tsx

```diff
@@ -203,26 +253,44 @@ export function ExperimentalAutoForm<M extends AbstractModel>({
   }
 
   return (
-    <VerticalLayout className="auto-form" theme="spacing" data-testid="auto-form">
+    <div className="auto-form" data-testid="auto-form">
       <VerticalLayout className="auto-form-fields">
         {layout}
         {formError ? <div style={{ color: 'var(--lumo-error-color)' }}>{formError}</div> : <></>}
       </VerticalLayout>
-      <HorizontalLayout className="auto-form-toolbar" theme="spacing">
-        {form.dirty ? (
-          <Button theme="tertiary" onClick={() => form.reset()}>
-            Discard
-          </Button>
-        ) : null}
+      <div className="auto-form-toolbar">
```

Comment:
The buttons here are rendered in the reverse order than they will be displayed. This has the benefit that when using keyboard navigation, the first button that gets focused after the fields is the save button, then the discard button, and finally the delete button. Visually the delete button is aligned to the left, and the save and discard buttons on the right. This basically reproduces how `vaadin-crud` renders and aligns the buttons.